src/gsy_e/models/area/scm_manager.py

```diff
@@ -448,97 +526,94 @@ def calculate_community_after_meter_data(self):
             self.community_data.self_consumed_energy_kWh += (
                 home_data.self_production_for_community_kWh)
 
-    def calculate_home_energy_bills(
-            self, home_uuid: str) -> None:
-        """Calculate energy bills for one home."""
-        assert home_uuid in self._home_data
-
-        home_data = self._home_data[home_uuid]
-
-        market_maker_rate = home_data.market_maker_rate
-        grid_fees = home_data.grid_fees
-        taxes_surcharges = home_data.taxes_surcharges
-        feed_in_tariff = home_data.feed_in_tariff
-
+    def _init_area_energy_rates(self, home_data: HomeAfterMeterData) -> AreaEnergyRates:
         slots_per_month = (duration(days=1) / GlobalConfig.slot_length) * monthrange(
             self._time_slot.year, self._time_slot.month)[1]
-        marketplace_fee = home_data.marketplace_monthly_fee / slots_per_month
-        assistance_fee = home_data.assistance_monthly_fee / slots_per_month
-        fixed_fee = home_data.fixed_monthly_fee / slots_per_month
 
-        intracommunity_base_rate_eur = (
-            market_maker_rate
+        intracommunity_base_rate = (
+            home_data.market_maker_rate
             if self._intracommunity_base_rate_eur is None
             else self._intracommunity_base_rate_eur)
-        market_maker_rate_decreased_fees = (
-                intracommunity_base_rate_eur
-                + grid_fees * (1.0 - self._grid_fees_reduction)
-                + taxes_surcharges)
-        market_maker_rate_normal_fees = market_maker_rate + grid_fees + taxes_surcharges
 
+        area_fees = AreaFees(
+            grid_fee=home_data.fee_properties.get(
```

Comment:
I am tempted to suggest to change the fee_properties from a Dict to a separate class, in order to be able to provide helper methods that generate these dicts, however feel free to omit it, this will work for now.

Reply:
Let me do this actually, we can also omit the validation on the Coefficient area then (or simplify it).

Reply:
Done: [88dd3ae](https://github.com/gridsingularity/gsy-e/pull/1756/commits/88dd3ae8a0bfed15b1ebaef3f96343f6158add2e)

I also took the liberty to move all dataclasses into its own module for SCM in order to reduce the length of the scm_manager module and to not have to scroll so much when working inside of the module, even though this will break the git blame chain of most of the scm dataclasses: https://github.com/gridsingularity/gsy-e/pull/1756/commits/e290d75c2ad318d3afd067f9549214b697acf600
What do you think?

Reply:
You did well and I was thinking about that too in the past, the module was already large, thanks!